server/src/main/java/io/crate/execution/engine/pipeline/SimpleLimitAndOffsetProjector.java

```diff
@@ -27,12 +27,12 @@
 import io.crate.data.Row;
 import io.crate.data.SkippingBatchIterator;
 
-public class SimpleTopNProjector implements Projector {
+public class SimpleLimitAndOffsetProjector implements Projector {
```

Comment:
```suggestion
public class LimitAndOffsetProjector implements Projector {
```

There is no `ComplexLimitAndOffsetProjector` the Simple prefix is not really adding meaningful information.